webtau-open-api/src/main/java/com/twosigma/webtau/openapi/OpenApiSpecValidator.java

```diff
@@ -50,15 +50,24 @@ public void validateApiSpec(HttpValidationResult result) {
             return;
         }
 
+        String relativePath = extractPath(result.getFullUrl());
+
+        SimpleRequest.Builder builder = new SimpleRequest.Builder(result.getRequestMethod(), relativePath);
+        if (result.getRequestContent() != null) {
+            builder.withBody(result.getRequestContent());
+        }
+        if (result.getRequestHeader() != null) {
+            result.getRequestHeader().forEachProperty(builder::withHeader);
+        }
+
+        SimpleRequest request = builder.build();
+
```

Comment:
method grew a bit. will we benefit from extracting code to two separate methods: buildRequest, buildResponse?